Testing/ORKTest/ORKTest/MainViewController.m

```diff
@@ -272,12 +273,12 @@ - (void)viewDidLoad {
                            @"Fitness Task",
                            @"GAIT Task",
                            @"Memory Game Task",
-                           @"PSAT Task",
-                           @"Reaction Time Task",
-                           @"Timed Walk Task",
+                           @"Two Finger Tapping Task",
                            @"Tone Audiometry Task",
+                           @"Reaction Time Task",
                            @"Tower Of Hanoi Task",
-                           @"Two Finger Tapping Task",
+                           @"PSAT Task",
+                           @"Timed Walk Task",
```

Comment:
@julientherier Can you take a look at `ORKTest` again? I didn't see `Hole Peg Test` in the list.